src/main/java/org/betonquest/betonquest/quest/condition/world/WorldCondition.java

```diff
@@ -0,0 +1,31 @@
+package org.betonquest.betonquest.quest.condition.world;
+
+import org.betonquest.betonquest.api.profiles.OnlineProfile;
+import org.betonquest.betonquest.api.quest.condition.online.OnlineCondition;
+import org.betonquest.betonquest.exceptions.QuestRuntimeException;
+import org.bukkit.World;
+
+/**
+ * A condition that checks if the player is in a specific world.
+ */
+public class WorldCondition implements OnlineCondition {
+
+    /**
+     * The world to check.
+     */
+    private final World world;
+
+    /**
+     * Create a new World condition.
+     *
+     * @param world the world to check
+     */
+    public WorldCondition(final World world) {
```

Comment:
I would say that the World should be injected as Variable. May we need to talk about this in the voice chat, because there is more background knowledge.